schematic_api/api/routes.py

```diff
@@ -815,3 +815,10 @@ def get_nodes_display_names(schema_url: str, node_list: list[str]) -> list:
     node_display_names = gen.get_nodes_display_names(node_list, mm_graph)
     return node_display_names
 
+def get_schematic_version() -> str:
+    """
+    Return the current version of schematic
+    """
+    if "VERSION" in os.environ:
+        version = os.environ["VERSION"]
+    return version
```

Comment:
If users use this locally, I think it would make sense to provide a more specific error message so that users know that this endpoint can only be used within a docker container. The current message: "UnboundLocalError: local variable 'version' referenced before assignment" is confusing for users

Reply:
That makes sense